### DISCUSSION TITLE
Which are the best events: in-person or virtual?

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#196] [General]
InnerSource Commons Summits used to all be in-person - and we switched to virtual in 2020 to respond to the circumstances of that time.

In 2022 and 2023 we continued to keep our annual summit as a virtual event for the following reasons:
- **Accessibility**: we are a growing community and wanted to make sure that folks don't have to have a travel budget to participate in the summit. This is also based on the assumption that hybrid events favour in-person attendees, and it's hard to make virtual attendees feel like they have had an equal experience. 
- **Cost and risk management**: We also have to acknowledge that the cost of physical events, and the event logistics are much greater than that of a virtual event. Plus - the requirement for upfront payments for physical events can introduce financial risks (and indeed health risks) in these more uncertain times. 

However, in saying that - we all really feel the need to get back to face to face meetings because of:
- **Strong bonds**: there is nothing like meeting people face to face to build the strong relationships that are at the heart of great communities. 
- **Deep work**: there are some discussion and activities that are very hard to conduct in a virtual environment - in-person workshops still work better. 

As a result, we decided to:
- Keep our summit virtual for maximum reach and equal experience for virtual attendees. 
- Introduce the idea of an InnerSource Gathering for smaller in-person meetings focusing on maximizing time together
  - focus on workshops, not 1:many presentations
  - smaller groupings
  - first one was held in conjunction with larger event (to save travel budgets/justification)

We regularly review this decision over in the Marketing Working Group in the ISC Slack. If you have insights or a point of view that can help us decide plans fo the future  - please add them to this discussion.

**Comment by claredillon** [comment]
@MaineC pointed us at this resource - an interesting point of information: 

https://remotesynthesis.com/blog/developer-conferences/ ... Some thoughts on conferences post pandemic. If true for our community as well, this supports out split of events depending on purpose (learning/ outreach Vs pure networking

**Reply by claredillon** [reply]
Thanks for sharing this @mainec - it also helps folks understand the economics of events, which are often a big factor in the decision-making around this. Most of the foundations and communities who put on large in-person events often have full time event management employees because the logistics are so much more challenging than virtual events. 

I'll also note that because of the current economic circumstances, many of the orgs who used to do event sponsorship are tightening purse strings (marketing budgets) and it has become harder to convince orgs to sponsor events. We did have some people sponsor from HR/recruitment budgets during the pandemic - but all those advertising budgets seem to have been cut somewhere in recent months as talks of recession for the sector loom larger. 

This may be all temporary - but while they are impacting things, I think it is the right decision to continue to be looking for lower-cost (both money and time) and lower-risk options for facilitating in-person gatherings...

**Reply by MaineC** [reply]
Those hiring budgets likely also were cut in particular for orgs with hiring freezes in place...
